nftables.nix

```diff
@@ -1,4 +1,4 @@
-{ config, pkgs, lib, ... }:
+{ config, ... }:
 let cfg = config.pi-router.interfaces; in
 {
 
```

Comment:
Can we maybe add a comment here, including the command to build the nftables config?

Reply:
Added command. Not pretty but it works.